### PR TITLE
Fix #537: Add service to find inbox message just by ID

### LINKED ISSUE #537: Add service to find inbox message just by ID

### PR DESCRIPTION
**Pull request by petrdvorak**

### CODE REVIEW

File: powerauth-push-server/src/main/java/io/getlime/push/service/InboxService.java [hunk 1]
```diff
@@ -88,6 +88,15 @@ public GetInboxMessageDetailResponse fetchMessageDetail(String userId, String ap
         return inboxMessageConverter.convertResponse(messageEntity.get());
     }
 
+    @Transactional(readOnly=true)
+    public GetInboxMessageDetailResponse fetchMessageDetail(String messageId) throws InboxMessageNotFoundException, AppNotFoundException {
+        final Optional<InboxMessageEntity> messageEntity = inboxRepository.findById(messageId);
+        if (!messageEntity.isPresent()) {
+            throw new InboxMessageNotFoundException("Unable to fetch message: " + messageId + ".");
+        }
+        return inboxMessageConverter.convertResponse(messageEntity.get());
```
Comment (banterCZ) [inline powerauth-push-server/src/main/java/io/getlime/push/service/InboxService.java:97]: Personally, I prefer `orElseThrow` approach.
```suggestion
        final InboxMessageEntity messageEntity = inboxRepository.findById(messageId)..orElseThrow(() ->
                new InboxMessageNotFoundException("Unable to fetch message: " + messageId + "."));
        return inboxMessageConverter.convertResponse(messageEntity);
```

**Review by banterCZ — APPROVED** [review]
Looks good.

**Review by romanstrobl — APPROVED** [review]
Looks good.

File: powerauth-push-server/src/main/java/io/getlime/push/service/InboxService.java (hunk 1 above)
Comment (petrdvorak) [reply powerauth-push-server/src/main/java/io/getlime/push/service/InboxService.java:97]: We currently do not use this in the code - I will leave it as is for now.

**Pull request merged**